### PR TITLE
[WIP] Switched to SwiftUI

### PR DESCRIPTION
**Pull request by ziccardi**
## Motivation
- Gui didn't work well on iPad
- Gui had problems with dark mode
- Gui wasn't using SwuiftUI

**This will be moved to ready state when the new AerogearPush-Swift package version will be published**

Blocked by https://github.com/aerogear/aerogear-ios-push/pull/117

### CODE REVIEW

File: UnifiedPushHelloWorld/HelloWorldSwift/ContentView.swift [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import SwiftUI
+
+struct ContentView: View {
+    @ObservedObject var notificationListModel = ListViewModel()
+    var items = ["a1", "a2"]
```
Comment (secondsun) [inline UnifiedPushHelloWorld/HelloWorldSwift/ContentView.swift]: Is this variable used?

File: UnifiedPushHelloWorld/HelloWorldSwift/SceneDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        if let windowScene = scene as? UIWindowScene {
+            let window = UIWindow(windowScene: windowScene)
+            window.rootViewController = UIHostingController(rootView: contentView)
+            self.window = window
+            window.makeKeyAndVisible()
+        }
+    }
+
+    func sceneDidDisconnect(_ scene: UIScene) {
+        // Called as the scene is being released by the system.
```
Comment (secondsun) [inline UnifiedPushHelloWorld/HelloWorldSwift/SceneDelegate.swift]: Heya, I see several empty method bodies, are these supposed to have any functionality in them?  Can the be removed?  Maybe a comment "intentionally left blank"?

File: UnifiedPushHelloWorld/HelloWorldSwiftUITests/HelloWorldSwiftUITests.swift [hunk 3]
```diff
@@ -0,0 +1,52 @@
+/*
```
Comment (secondsun) [inline UnifiedPushHelloWorld/HelloWorldSwiftUITests/HelloWorldSwiftUITests.swift:1]: This test class doesn't seem to test anything, maybe remove it or add tests?

**Review by secondsun — CHANGES_REQUESTED** [review]
Heya, a few questions after a quick skim

File: UnifiedPushHelloWorld/HelloWorldSwift/ContentView.swift (hunk 1 above)
Comment (ziccardi) [reply UnifiedPushHelloWorld/HelloWorldSwift/ContentView.swift]: Good catch. Forgot to delete that one!

File: UnifiedPushHelloWorld/HelloWorldSwiftUITests/HelloWorldSwiftUITests.swift (hunk 3 above)
Comment (ziccardi) [reply UnifiedPushHelloWorld/HelloWorldSwiftUITests/HelloWorldSwiftUITests.swift:1]: Yup. It is a placeholder. We will populate the tests in a future PR.

**Comment by ziccardi** [comment]
ping @Aindriunannerb

**Review by secondsun — APPROVED** [review]

**Pull request closed without merging**